### DISCUSSION TITLE
Format a number with commas to separate thousands in Python

### DISCUSSION THREAD

**Discussion by snurulazizah** [teamalgoritma/community#153] [Q&A] [answered]
Bagaimana membuat format number dengan koma pada angka ribuan di Python?

**Comment by yosiaazarya** [comment, marked as answer]
Ada beberapa cara yang dapat digunakan,
Cara yang pertama adalah dengan menerapkan pengaturan display pada pandas: 
`pd.options.display.float_format = '{:,}'.format`

Cara yang kedua adalah dengan meng-aplikasikan formating pada kolom terkait: 
`df['Value'] = df.apply(lambda x: "{:,}".format(x['Value']), axis=1)`
